### PR TITLE
migrate to jitpack

### PR DESCRIPTION
**Pull request by aristotelism**
* Migrate from bintray to jitpack
* Updated gradle/android gradle plugin/android sdk version.

### CODE REVIEW

File: android-radar/build.gradle [hunk 1]
```diff
@@ -53,7 +30,33 @@ dependencies {
...
+                        license {
+                            name = 'The MIT License (MIT)'
+                            url = 'http://opensource.org/licenses/MIT'
+                        }
+                    }
+                    developers {
+                        developer {
+                            id = 'cedexis'
+                            name = 'Cedexis'
+                            email = 'support@cedexis.com'
```
Comment (ikalligeros) [inline android-radar/build.gradle:51]: I wonder if this email is still valid and if anyone is monitoring it.

File: android-radar/build.gradle [hunk 2]
```diff
@@ -53,7 +30,33 @@ dependencies {
...
-version = libraryVersion
-apply from: 'https://raw.githubusercontent.com/cedexis/JCenter/master/installv1.gradle'
-apply from: 'https://raw.githubusercontent.com/cedexis/JCenter/master/bintrayv1.gradle'
\ No newline at end of file
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                pom {
+                    name = 'Android Radar Library'
```
Comment (ikalligeros) [inline android-radar/build.gradle:38]: Maybe it's a good opportunity for re-branding: "Citrix ITM Android Radar Library"

**Comment by ikalligeros** [comment]
Changes look good! In general my only comment is to change anything related to Cedexis to Citrix ITM.

File: android-radar/build.gradle (hunk 2 above)
Comment (aristotelism) [reply android-radar/build.gradle:38]: Thanks @ikalligeros I will ask Kostas if this is ok. If it requires approval from others, I leave it as is.

File: android-radar/build.gradle (hunk 1 above)
Comment (aristotelism) [reply android-radar/build.gradle:51]: @ikalligeros Yes this still replies, it redirects the sender to citrix support.

File: app/src/main/java/com/cedexis/simpleradardemo/MainActivity.java [hunk 3]
```diff
@@ -2,13 +2,13 @@
...
 import android.os.Bundle;
 import android.os.StrictMode;
-import android.support.v7.app.AppCompatActivity;
 import android.util.Log;
 import android.view.View;
 import android.view.ViewGroup;
-import android.webkit.WebView;
 import android.widget.Button;
 
+import androidx.appcompat.app.AppCompatActivity;
```
[the remark this answers is not in the document]
Comment (aristotelism) [reply app/src/main/java/com/cedexis/simpleradardemo/MainActivity.java:10]: @chris7ceid I did manually.

**Review by chris7ceid — APPROVED** [review]

File: android-radar/build.gradle (hunk 2 above)
Comment (aristotelism) [reply android-radar/build.gradle]: Ok I changed the name. I will handle the code rebranding in the next release.

File: android-radar/build.gradle (hunk 1 above)
Comment (aristotelism) [reply android-radar/build.gradle]: I have removed for now the email, in lack of a good alternative. Users can submit tickets for now.

**Pull request merged**